src/olympia/users/models.py

```diff
@@ -142,7 +142,9 @@ class UserProfile(OnChangeMixin, ModelBase, AbstractBaseUser):
     failed_login_attempts = models.PositiveIntegerField(default=0,
                                                         editable=False)
 
-    is_verified = models.BooleanField(default=True)
+    # Is the profile page for this account publicly viewable?
+    # Note: this isn't applicable to legacy frontend pages.
```

Comment:
This needs a bit more explanation imho, why it's not or what this means for legacy frontend pages.